server/src/assetComputeDevTool.js

```diff
@@ -139,6 +139,11 @@ class AssetComputeDevTool {
         );
     }
 
+    async isJournalReady() {
+        return this.assetCompute.isEventJournalReady();
+        return isReady;
```

Comment:
also, delete this line (i couldn't figure out how to get the suggestion tool to let me suggest multiple line changes at once.)

Reply:
👍  I tried the suggestion tools -> `commit suggestion` and it messed it up. thanks for pointing it out.